modulars/league_stats.rb

```diff
@@ -51,15 +51,15 @@ def lowest_scoring_home_team
 
```

Comment:
These look this way because I changed the expected in the test when I switched to dummy files